README.md

```diff
@@ -22,6 +22,19 @@ extensible to working with different manifest schemas and PKI tooling.
 
 [Official Documentation](https://docs.sigstore.dev/rekor/overview).
 
+## Current State of Rekor v1
+
+Rekor v1 is in maintenance mode.
+
+We are actively developing a new version of Rekor designed to be easy to maintain and cheaper to operate.
+Building on the active development in the Certificate Transparency ecosystem, Rekor v2 will be backed by a
+[tile-based log](https://transparency.dev/articles/tile-based-logs/) and will use a modernized version of Trillian,
+[Trillian-Tessera](https://github.com/transparency-dev/trillian-tessera).
+
+Follow its progress on the [rekor-tiles](https://github.com/sigstore/rekor-tiles/) repo, and learn more about
```

Comment:
should you say something about the plan to merge code back into this repo or that `rekor-tiles` will be long-lived?

Reply:
Updated. I'm planning to merge back in at the moment.